acceptance/repository-cloudant/src/__tests__/cloudant-default-repository.acceptance.ts

```diff
@@ -0,0 +1,20 @@
+// Copyright IBM Corp. 2019. All Rights Reserved.
+// Node module: @loopback/test-repository-cloudant
+// This file is licensed under the MIT License.
+// License text available at https://opensource.org/licenses/MIT
+
+import {DefaultTransactionalRepository} from '@loopback/repository';
+import {
+  CrudRepositoryCtor,
+  crudRepositoryTestSuite,
+} from '@loopback/repository-tests';
+import {CLOUDANT_CONFIG, CLOUDANT_FEATURES} from './cloudant.datasource';
+
+describe('CLOUDANT + DefaultTransactionalRepository', () => {
```

Comment:
Since Cloudant does not support transactions, let's use `DefaultCrudRepository` as we already do in MongoDB tests.

https://github.com/strongloop/loopback-next/blob/9def622d7bafb7eb2470cd896372f030998e3fd4/acceptance/repository-mongodb/src/__tests__/mongodb-default-repository.acceptance.ts#L6-L20